JPS_Ontology/ontology/ontoisochrone/OntoIsochrone.csv

```diff
@@ -0,0 +1,24 @@
+Cycle,Class,TransportMode,IS-A,,,,Cycle describes the act of travelling via a bicycle.,https://www.theworldavatar.com/kg/ontoisochrone/,Cycle
+Walk,Class,TransportMode,IS-A,,,,Walk describes the act of traveling on foot. ,https://www.theworldavatar.com/kg/ontoisochrone/,Walk
+RoadConditions,Class,,,,,,Road conditions describes the conditions of the road. ,https://www.theworldavatar.com/kg/ontoisochrone/,Road Conditions
+Flooded,Class,RoadConditions,IS-A,,,,Road conditions under flooded circumstances.,https://www.theworldavatar.com/kg/ontoisochrone/,Flooded
+Normal,Class,RoadConditions,IS-A,,,,Road conditions under normal circumstances.,https://www.theworldavatar.com/kg/ontoisochrone/,Normal
+hasTimeThreshold,Data Property,,,Isochrone,https://www.w3.org/2006/time#minute,,A relationship between an isochrone and its time threshold.,https://www.theworldavatar.com/kg/ontoisochrone/,has time threshold
+hasRoadConditions,Object Property,,,Isochrone,RoadConditions,,A relationship that describes the road condition of an isochrone.,https://www.theworldavatar.com/kg/ontoisochrone/,has road conditions
+hasTotalPopulation,Data Property,,,Isochrone,integer,,A relationship between an isochrone and its covered total population number.,https://www.theworldavatar.com/kg/ontoisochrone/,has total population
+hasMenPopulation,Data Property,,,Isochrone,integer,,A relationship between an isochrone and its covered men population number.,https://www.theworldavatar.com/kg/ontoisochrone/,has men population
+hasWomenPopulation,Data Property,,,Isochrone,integer,,A relationship between an isochrone and its covered women population number.,https://www.theworldavatar.com/kg/ontoisochrone/,has women population
+hasWomenofReproductiveAge,Data Property,,,Isochrone,integer,,A relationship between an isochrone and its covered women of reproductive age (age 15 - 49) population number.,https://www.theworldavatar.com/kg/ontoisochrone/,has women of reproductive age population
+hasChildrenPopulation,Data Property,,,Isochrone,integer,,A relationship between an isochrone and its covered children (age under 5) population number.,https://www.theworldavatar.com/kg/ontoisochrone/,has children population
+hasYouthPopulation,Data Property,,,Isochrone,integer,,A relationship between an isochrone and its covered youth (age 15 - 24) population number.,https://www.theworldavatar.com/kg/ontoisochrone/,has youth population
+hasElderlyPopulation,Data Property,,,Isochrone,integer,,A relationship between an isochrone and its covered elderly (age above 60) population number.,https://www.theworldavatar.com/kg/ontoisochrone/,has elderly population
+isCalculatedUsing,Object Property,,,Isochrone,TransportMode,,A relationship between an isochrone and the transport mode it is used for isochrone calculations.,https://www.theworldavatar.com/kg/ontoisochrone/,is calculated using
```

Comment:
The name of this relationship is actually a bit misleading. How about `assumesTransportMode`?

"it is used" -> "that is used"